bitcanna/chain.json

```diff
@@ -579,6 +588,12 @@
       "kind": "Indonode",
       "url": "https://explorer.indonode.net/bitcanna",
       "tx_page": "https://explorer.indonode.net/bitcanna/tx/${txHash}"
+    },
+    {
+      "kind": "Safe Block",
+      "url": "https://explorer.safeblock.space/bitcanna",
+      "tx_page": "https://explorer.safeblock.space/bitcanna/tx/${txHash}"
```

Comment:
I think you're missing a comma here, causing the validation to fail

Reply:
Many thanks, a small mistake on my part. I'm sorry.